ndless-sdk/toolchain/build_toolchain.sh

```diff
@@ -16,7 +16,7 @@ set -eu
 
 TARGET=arm-none-eabi
 PREFIX="${PWD}/install" # or the directory where the toolchain should be installed in
-PARALLEL="${PARALLEL--j4}" # or "-j<number of build jobs>"
+PARALLEL="${PARALLEL--$(nproc --all)}" # or "-j<number of build jobs>"
```

Comment:
`nproc` isn't a thing on macOS (unless `coreutils` is installed, apparently), but from what I read, this should work:
`getconf _NPROCESSORS_ONLN`